### ISSUE TITLE
sympy function to display only certain number of terms per line for latex

### ISSUE THREAD

**Issue by jgoppert** [sympy/sympy#3001]
I have always been annoyed by having to manually wrap latex equations in my IPython notebooks, so I wrote this function. I think something like it should be included in sympy to simplify formatting of long equations.

``` python
import sympy
from IPython.display import Latex

def terms(lhs, expr,terms_per_line=1):
    """
    This function is written to handle displaying long latex equations.
    """
    latex = r'\begin{align*}' + '\n'
    terms = expr.as_ordered_terms()
    n_terms = len(terms)
    term_count = 1
    for i in range(n_terms):
        term = terms[i]
        term_start = r''
        term_end = r''
        sign = r'+'
        if term_count > terms_per_line:
            term_start = '&'
            term_count = 1
        if term_count == terms_per_line:
            term_end = r'\ldots \\' + '\n'
        if term.as_ordered_factors()[0]==-1:
            term = -1*term
            sign = r'-'
        if i == 0: # beginning
            latex += r'{:s} =& {:s} {:s} {:s}'.format(sympy.latex(lhs),
                                                     term_start,sympy.latex(term),term_end)
        elif i == n_terms-1: # end
            latex += r'{:s} {:s} {:s} {:s}'.format(term_start,sign,sympy.latex(term),term_end)
        else: # middle
            latex += r'{:s} {:s} {:s} {:s}'.format(term_start,sign,sympy.latex(term),term_end)
        term_count += 1
    latex += r'\end{align*}' + '\n'
    return Latex(latex)
```

**Comment by rajeev2010** [comment]
i have recently found a bug in the above function. if the very first term has a negative sign it does not get displayed. it can be easily remedied by the following change -
```python
        if i == 0: # beginning
            if sign == r'+': sign = r""
            latex += r'{:s} =& {:s} {:s} {:s} {:s}'.format(sympy.latex(lhs),
                                                     term_start,sign,sympy.latex(term),term_end)
```
@jgoppert thanks for sharing this piece of code. it has helped me avoid manually processing latex expressions generated by sympy. i hope this bug fix will be helpful to others as well.

**Comment by oscarbenjamin** [comment]
#15944 will add something based off this.

**Comment by oscargus** [comment]
I have merged #15944 so I believe that this can be closed. Thanks for the code!

**Comment by jgoppert** [comment]
Thanks!
